src/Directory.build.targets

```diff
@@ -26,6 +26,12 @@
     <TargetPlatformVersion>10.0.16299.0</TargetPlatformVersion>
     <DefineConstants>$(DefineConstants);NETFX_CORE;XAML;WINDOWS_UWP</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup Condition="$(TargetFramework.StartsWith('net5.0-windows10.0'))">
+    <TargetPlatformVersion>10.0.19041.0</TargetPlatformVersion>
+  </PropertyGroup>
+  <PropertyGroup Condition="$(TargetFramework.StartsWith('net6.0-windows10.0'))">
```

Comment:
This belongs with the project that targets windows stuff

Reply:
Removed